src/leo_gateway_http_commons.erl

```diff
@@ -738,30 +738,80 @@ get_range_object(Req, Bucket, Key, {error, badarg}) ->
     ?reply_bad_range([?SERVER_HEADER], Key, <<>>, Req);
 get_range_object(Req, Bucket, Key, {_Unit, Range}) when is_list(Range) ->
     Mime = leo_mime:guess_mime(Key),
-    Header = [?SERVER_HEADER,
-              {?HTTP_HEAD_RESP_CONTENT_TYPE,  Mime}],
-    {ok, Req2} = cowboy_req:chunked_reply(?HTTP_ST_PARTIAL_CONTENT, Header, Req),
-    get_range_object_1(Req2, Bucket, Key, Range, undefined).
+    case get_body_length_1(Key, Range) of
+        {ok, Length} ->
+            Header = [?SERVER_HEADER,
+                      {?HTTP_HEAD_RESP_CONTENT_TYPE,  Mime},
+                      {?HTTP_HEAD_RESP_CONTENT_LENGTH,integer_to_list(Length)}],
+            Req2 = cowboy_req:set_resp_body_fun(
+                     fun(Socket, Transport) ->
+                             get_range_object_1(Req, Bucket, Key, Range, undefined, Socket, Transport)
+                     end, 
+                     Req),
```

Comment:
@windkit I found out why CI failed.
You need to use set_resp_body_fun's another version.

https://github.com/ninenines/cowboy/blob/1.0.0/src/cowboy_req.erl#L776

Try to pass the content length you calculated as the first argument.

Since R16B03-1 coudn't understand transfer-encoding: identify, this error have occured on CI.
https://github.com/erlang/otp/blob/OTP_R16B03-1/lib/inets/src/http_client/httpc_handler.erl#L1122

The latest one can understand this header.
https://github.com/erlang/otp/blob/OTP-17.5.6/lib/inets/src/http_client/httpc_handler.erl#L1130